pandas/tests/series/test_apply.py

```diff
@@ -124,7 +124,7 @@ def test_apply_datetimetz(self):
 
         # change dtype
         result = s.apply(lambda x: x.hour)
-        exp = pd.Series(list(range(24)) + [0], name='XX', dtype=np.int32)
+        exp = pd.Series(list(range(24)) + [0], name='XX', dtype=np.int64)
```

Comment:
hmm, I don't think this should have changed, these are normally ``int32``s

any idea? (also this might not be the same on windows)